3_rest_requests_multi_model.ipynb

```diff
@@ -97,6 +97,74 @@
     "else:\n",
     "    print('not fraud')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5697c2ff",
+   "metadata": {},
+   "source": [
+    "## Example 1: user buys a coffee\n",
+    "\n",
+    "In this example, the user is buying a coffee. The parameters given to the model are:\n",
+    "* same location as the last transaction (distance=0)\n",
+    "* same median price (ratio_to_median=1)\n",
+    "* using pin number (pin=1)\n",
+    "* using chip (chip=1)\n",
+    "* no online transaction (online=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0393a5a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = [0.0, 1.0, 1.0, 1.0, 0.0]\n",
+    "prediction = rest_request(data)\n",
+    "prediction\n",
+    "threshhold = 0.995\n",
+    "\n",
+    "if (prediction[0] > threshhold):\n",
+    "    print('The model thinks this is fraud')\n",
+    "else:\n",
+    "    print('The model thinks this is not fraud')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e889cdd6",
+   "metadata": {},
+   "source": [
+    "## Example 2: fraudulent transaction\n",
+    "\n",
+    "In this example, someone stole the user credit card and is buying something online. The parameters given to the model are:\n",
+    "* very far away from latest transaction (distance=100)\n",
+    "* similar median price (ratio_to_median=1.2)\n",
+    "* not using chip (chip=0)\n",
+    "* not using pin number (pin=0)\n",
+    "* online transaction (online=1)"
+   ]
```

Comment:
some recommended text changes shown here in bold:
* very far away from **the last** transaction
* median price **similar to the last transaction**
* not using **a** pin number
* not using the **credit card** chip 
 * **is an** online transaction